frontend/src/index.js

```diff
@@ -18,15 +17,15 @@ posthog.init(API_KEY, {
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <PostHogProvider client={posthog}>
-      <SocketProvider>
-        <LazyLoader
-          loader={<GenericLoader />}
-          component={() => import("./App.jsx")}
-          componentName="App"
-        />
-      </SocketProvider>
-    </PostHogProvider>
-  </React.StrictMode>
+  // <React.StrictMode>
```

Comment:
Why it commented?

Reply:
No suppose to commented. I'll make that change